apps/app_mixmonitor.c

```diff
@@ -1311,6 +1330,14 @@ static int mixmonitor_exec(struct ast_channel *chan, const char *data)
 			filename_read = ast_strdupa(filename_parse(opts[OPT_ARG_READNAME], filename_buffer, sizeof(filename_buffer)));
 		}
 
+		if (ast_test_flag(&flags, MUXFLAG_BEEP_START)) {
+			start_sound = ast_strdupa(S_OR(opts[OPT_ARG_START_SOUND], "beep"));
```

Comment:
This can be:

```
start_sound = S_OR(ast_skip_blanks(opts[OPT_ARG_START_SOUND]), "beep");
```

Which avoids the `ast_strdupa()` and avoids empty filenames too.